authors/settings.py

```diff
@@ -83,10 +84,10 @@
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.postgresql_psycopg2',
-        'NAME': os.environ.get('DATABASE_NAME'),
-        'USER': os.environ.get('DATABASE_USER'),
-        'PASSWORD': os.environ.get('DATABASE_PASSWORD'),
-        'HOST': os.environ.get('DATABASE_HOST'),
+        'NAME':'authors',
+        'USER': 'postgres',
+        'PASSWORD':'',
+        # 'HOST': os.environ.get('5432'),
     }
```

Comment:
@NicholusMuwonge. You shouldn't be exposing sensitive data. Consider using environment variables.

Reply:
This I am Trying to implement but still not functioning. Let me fix it